BSP/Simulator/Src/BSP_ADC.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "BSP_ADC.h"
+#include "common.h"
+#include "config.h"
+
+/*
+C file that reads the raw ADC data from the CSV file, 
+converts it back to a distance pressed (percentage for now), and stores both.
+We should be able to request both the raw ADC data 
+and the distance (percentage for now) from this library.
+ */
+
+//NOTE: ADC precision is 12 bits (4096), range 3.3V
+//I'm assuming 3.3V corresponds to a 100% pressed pedal
+
+#define FILE_NAME "BSP/Simulator/Hardware/Data/Pedals.csv"
+
+int ADCvalueAccel, ADCValueBrake;
+float ADCPercentageAccel, ADCPercentageBrake;
+
+void ADC_Init(void) {
+    // Checking if file exists
+    if(access(FILE_NAME, F_OK) == -1){
```

Comment:
I would change this to a != 0 just to be more inclusive. Honestly, I'm not sure what all values it will return, but that just makes it more inclusive.